Vegeting/Screens/CreateGroup/ViewController/LocationSearchingViewController.swift

```diff
@@ -202,19 +204,29 @@ final class LocationSearchingViewController: UIViewController {
 
 extension LocationSearchingViewController: UITableViewDataSource {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return addressResultList.count + placeResultList.count
+        let totalCount = addressResultList.count + placeResultList.count
+        if totalCount == 0 {
+            return autoSearchResults.count
+        } else {
+            return addressResultList.count + placeResultList.count
+        }
```

Comment:
```suggestion
        return totalCount == 0 ? autoSearchResults.count : totalCount
```
p2;
삼항연산자를 쓰게 되면 한줄로 간단하게 해결이 가능합니다.

Reply:
오 훨씬 깔끔하네요 !!! [d81e9b3](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/93/commits/d81e9b367fc6aa72ca0e2d66cdc973ef871fb8ba)에서반영하였습니다. 삼항연산자 잘쓰면 좋군요!